test/functional/cmdLineTests/sigabrtHandlingTest/sigabrtHandlingTest.xml

```diff
@@ -33,66 +33,83 @@
    This should be sufficient to test the -XX:[+|-]HandleSIGABRT option.
  -->
  <variable name="DUMP" value="-Xdump:system+heap+snap+jit:none" />
+ <variable name="LIMT_HEAPSIZE" value="-Xmx20m" />
 
  <test id="Default">
-  <command>$EXE$ $DUMP$ $CP$ abort</command>
+  <command>$EXE$ $LIMT_HEAPSIZE$ $DUMP$ $CP$ abort</command>
   <output type="success" regex="no">Processing dump event "abort"</output>
   <output type="required" regex="no">Processed dump event "abort"</output>
  </test>
 
  <test id="-XX:+HandleSIGABRT">
-  <command>$EXE$ $DUMP$ -XX:+HandleSIGABRT $CP$ abort</command>
+  <command>$EXE$ $LIMT_HEAPSIZE$ $DUMP$ -XX:+HandleSIGABRT $CP$ abort</command>
   <output type="success" regex="no">Processing dump event "abort"</output>
   <output type="required" regex="no">Processed dump event "abort"</output>
  </test>
 
  <test id="-XX:-HandleSIGABRT -XX:+HandleSIGABRT">
-  <command>$EXE$ $DUMP$ -XX:-HandleSIGABRT -XX:+HandleSIGABRT $CP$ abort</command>
+  <command>$EXE$ $LIMT_HEAPSIZE$ $DUMP$ -XX:-HandleSIGABRT -XX:+HandleSIGABRT $CP$ abort</command>
   <output type="success" regex="no">Processing dump event "abort"</output>
   <output type="required" regex="no">Processed dump event "abort"</output>
  </test>
 
  <test id="-XX:-HandleSIGABRT">
-  <command>$EXE$ $DUMP$ -XX:-HandleSIGABRT $CP$ abort</command>
+  <command>$EXE$ $LIMT_HEAPSIZE$ $DUMP$ -XX:-HandleSIGABRT $CP$ abort</command>
   <output type="success" regex="no">Invoking abort!</output>
   <output type="failure" regex="no">Processing dump event "abort"</output>
  </test>
+ 
+ <!-- Remove any core file created by the OS -->
+ <exec command="bash -c $Q$rm core.*$Q$" />
```

Comment:
I don't think you want the final `$` here. I think it should be:
```
  <exec command="bash -c $Q$rm core.*$Q" />
```
Same goes for several copies of this pattern below.

Reply:
It doesn't work without it
```
The variable "null" with value "bash -c $Q$rm core.*$Q" cannot be resolved because it is missing at least one token signifying the end of a variable ('$' or '}'); the first unbalanced start token is a '$'
```

Reply:
I forgot who was interpreting the `$var$` sequences (I thought it was a shell).